web/src/compose_ui.ts

```diff
@@ -14,39 +14,57 @@ import * as people from "./people";
 import * as popover_menus from "./popover_menus";
 import * as rtl from "./rtl";
 import * as stream_data from "./stream_data";
+import {user_settings} from "./user_settings";
 import * as user_status from "./user_status";
-import * as util from "./util";
+
+type messageType = "stream" | "private";
+type optsType = {
+    trigger: string;
+    private_message_recipient: string;
+    topic: string;
+    stream_id: number;
+};
+type selectionOffSelectedLines = {
+    before_lines: string;
+    separating_new_line_before: boolean;
+    selected_lines: string;
+    separating_new_line_after: boolean;
+    after_lines: string;
+};
```

Comment:
Type names should always start with a capital letter. I also don't like any of these names -- look at other TS files for example conventions to follow.